lib/matplotlib/backends/backend_ps.py

```diff
@@ -505,12 +505,12 @@ def set_font(self, fontname, fontsize, store=True):
                 self.fontname = fontname
                 self.fontsize = fontsize
 
-    def create_hatch(self, hatch):
+    def create_hatch(self, hatch, lw):
         sidelen = 72
         if hatch in self._hatches:
             return self._hatches[hatch]
         name = 'H%d' % len(self._hatches)
-        linewidth = mpl.rcParams['hatch.linewidth']
+        linewidth = lw
```

Comment:
Might as well drop the extra variable, though:
```suggestion
    def create_hatch(self, hatch, linewidth):
        sidelen = 72
        if hatch in self._hatches:
            return self._hatches[hatch]
        name = 'H%d' % len(self._hatches)
```